launch_ros/launch_ros/actions/lifecycle_node.py

```diff
@@ -28,11 +25,10 @@
 import lifecycle_msgs.msg
 import lifecycle_msgs.srv
 
+from .lifecycle_transition import LifecycleTransition
```

Comment:
`LifecycleTransition` doesn't exist in Humble: https://build.ros2.org/job/Hpr__launch_ros__ubuntu_jammy_amd64/21/testReport/junit/(root)/launch_ros/___/

Backporting it should be straightforward: #317